iroha/src/lib.rs

```diff
@@ -210,31 +213,33 @@ impl Iroha {
         let queue = Arc::clone(&self.queue);
         let tx_handle = task::spawn(async move {
             while let Some(transaction) = transactions_receiver.write().await.next().await {
+                log::info!("Pushing tx: {:?}", transaction);
```

Comment:
This would drastically increase messages in the console. If it was only used for debugging then remove it, otherwise please decrease log level and also write a more detailed log entry.

Reply:
Will remove it as soon as will fix sumeragi tests